docs/theme/Theme.mdx

```diff
@@ -24,13 +24,33 @@ export const App = () => (
   </ThemeProvider>
 );
 ```
-**Result:** In this example, the `darkTheme` from Reablocks is applied to your application. 
+**Result:** In this example, the `darkTheme` from Reablocks is applied to your application.
 Simply wrap your component hierarchy with the `ThemeProvider`, and the theme is automatically applied.
 
+### Create a reusable component with a custom theme
```

Comment:
We should call out there are 2 ways to use theme in reablocks. Also we should call out a user doesn't have to use tailwind if they don't want. @Shkils